CMakeLists.txt

```diff
@@ -46,8 +38,6 @@ include(PlatformConfiguration)
 include(CheckSSE)
 include(CheckAVX)
 
-option(BUILD_TESTS "Build unit tests" FALSE)
-
```

Comment:
Impressive "cleanup cmake" commit affects only these lines. Squash it with `cm_find_package` removal and put more information into the message

Reply:
The reason behind it is that there are 23 other PR and each of them has the same changes (not all of them are moving to nix CI for now). This split was to make them uniform.
Anyway, I squashed them now